ldp/nn/generation/base.py

```diff
@@ -0,0 +1,10 @@
+from abc import ABC, abstractmethod
+
+import torch
+from transformers import LogitsProcessor
+
+
+class LogitsProcessorWithFinalize(LogitsProcessor, ABC):
+    @abstractmethod
+    def finalize(self, token_ids: torch.Tensor) -> None:
+        pass
```

Comment:
Can you add a docstring here? It's customary in an `abstractmethod` to detail what subclassers should do.

Imo the signature alone isn't very intuitive, I think it needs a docstring